Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/README.md

```diff
@@ -78,43 +73,18 @@ Retrieve results for a file hash using WildFire
 | InfoFile.Info | string | Details of the report file. | 
 | InfoFile.Size | number | Size of the report file. | 
 | InfoFile.Type | string | The report file type. | 
+| File.FeedRelatedIndicators.value | String | Indicators that are associated with the File. | 
+| File.FeedRelatedIndicators.type | String | The type of the indicators that are associated with the File. | 
+| File.Tags | String | Tags that are associated with the File. | 
+| File.Behavior.details | String | File behavior details. | 
+| File.Behavior.action | String | File behavior action. | 
 
 
 #### Command Example
-```!file file=735bcfa56930d824f9091188eeaac2a1d68bc64a21f90a49c5ff836ed6ea723f```
-
-#### Context Example
-```json
-{
-    "DBotScore": [
-        {
-            "Indicator": "735bcfa56930d824f9091188eeaac2a1d68bc64a21f90a49c5ff836ed6ea723f",
-            "Score": 1,
-            "Type": "hash",
-            "Vendor": "WildFire"
-        },
-        {
-            "Indicator": "735bcfa56930d824f9091188eeaac2a1d68bc64a21f90a49c5ff836ed6ea723f",
-            "Score": 1,
-            "Type": "file",
-            "Vendor": "WildFire"
-        }
-    ],
-    "WildFire": {
-        "Report": {
-            "SHA256": "735bcfa56930d824f9091188eeaac2a1d68bc64a21f90a49c5ff836ed6ea723f",
-            "Status": "Success"
-        }
-    }
-}
-```
+``` ```
```

Comment:
Why no command example?

Reply:
Had an issue with `demisto-sdk generate-docs`, all the related comments are now fixed.